react/AsidedLayout/AsidedLayout.less

```diff
@@ -0,0 +1,21 @@
+@import (reference) "~seek-style-guide/theme";
+
+@media only screen and (min-width:740px) {
```

Comment:
Another magic number here.

In CR we use mixins to generate our media queries, probably not a huge difference but looks a little neater.